src/lib/FlightTasks/tasks/Orbit/FlightTaskOrbit_params.c

```diff
@@ -0,0 +1,87 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2019 PX4 Development Team. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ * 3. Neither the name PX4 nor the names of its contributors may be
+ *    used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+/**
+ * @file FlightTaksOrbit_params.c
+ * Orbit specific parameters.
+ *
+ * @author Dennis Mannhart <dennis.mannhart@gmail.com>
+ */
+
+/**
+ * Centripetal maximum acceleration
+ *
+ * @unit m/s/s
+ * @min 1.0
+ * @max 10.0
+ * @increment 0.5
+ * @decimal 1
+ * @group FlightTaskOrbit
+ */
+PARAM_DEFINE_FLOAT(ORBIT_ACC_MAX, 2.0f);
+
+/**
+ * Minimum radius of orbit
```

Comment:
FYI, are these to prevent someone shrinking a radius too low? Why does it have a maximum value of 10m?

Reply:
Yes, if the radius is <1m the vehicle doesn't do anything useful anymore and it can be confusing. I have no idea why the maximum is 10m. Probably the minimum radius should not be higher than the default radius of the circle QGC initially puts on the map for you to drag.